### PR TITLE
 movie info modal

### LINKED ISSUE #20: Movie Detail Modal
**Is your feature request related to a problem? Please describe.**
Display more information about a selected movie via modal. 

**Describe the solution you'd like**
Create a new SCSS to style the movie detail modal.

### PR DESCRIPTION
**Pull request by gonzalespaulb**
## Changes
1. Modal toggle on and off function
2. Modal styling
3. Modal animation

## Purpose
Display movie detail information  in a modal

Closes #20

### CODE REVIEW

File: movie-app/src/Components/MovieCard.js [hunk 1]
```diff
@@ -1,25 +1,36 @@
+import { useState } from 'react';
 
-import {useState} from 'react';
-
-const MovieCard = ({movie}) => {
+/**
+  detailClick: 1. Sends individual movie imdbID back to parent component(i.e. Searched) and fetch movie info by ID
+               2. toggleModal: open & close modal
+ */
```
Comment (Lap343) [inline movie-app/src/Components/MovieCard.js:6]: nice comments

**Review by Lap343 — APPROVED** [review]
Nice job guys. Lots of styling

**Review by krferrell — APPROVED** [review]
Everything is coming together! Looks great.

**Review by Gjhnsn — APPROVED** [review]
Looks good!

**Pull request merged**